### PR TITLE
add argo plugin

### PR DESCRIPTION
**Pull request by ganisback**
add argo workflow plugin component

## Summary by CodeRabbit

- **New Features**
	- Added a section for "Argo Workflow Configuration" in the documentation to assist with deployment.
	- Introduced an environment variable `ENABLE_ARGO_WORKFLOW` for optional installation of ARGO Workflow.
	- Included a new function to install ARGO Workflow in the Kubernetes initialization script.
	- Enhanced installation script to conditionally install ARGO Workflow based on environment variable.

- **Documentation**
	- Expanded "Prerequisites" and "Usage" sections in the README for clearer deployment instructions.
	- Added notes about disk space requirements and external port accessibility.

- **Bug Fixes**
	- Improved clarity and completeness of the documentation for better user experience.

### CODE REVIEW

File: docker-compose/README.md [hunk 1]
```diff
@@ -99,6 +99,10 @@ docker compose -f docker-compose.yml down -v
 
 Please refer to [Knative installation](https://opencsg.com/docs/csghub/101/helm/installation) for knative configuration.
 
+### Argo Workflow Configuration
+```
+kubectl apply -f https://ghp.ci/https://raw.githubusercontent.com/OpenCSGs/csghub-installer/refs/heads/main/docker-compose/argo/all.yaml
+```
```
Comment (wayneliu0019) [inline docker-compose/README.md:105]: 这块儿更改为下面这样吧
 ```
Please refer to [Knative installation](https://opencsg.com/docs/csghub/101/helm/installation) for argo workflow configuration.
```

File: helm-chart/install.sh [hunk 2]
```diff
@@ -20,7 +20,6 @@ fi
 
 ENABLE_K3S=${ENABLE_K3S:-"true"}
 ENABLE_DYNAMIC_PV=${ENABLE_DYNAMIC_PV:-"false"}
-ENABLE_KNATIVE_SERVING=${ENABLE_KNATIVE_SERVING:-$ENABLE_K3S}
```
Comment (wayneliu0019) [inline helm-chart/install.sh:23]: 这个为什么被删除了？ 手误吧？

Comment (ganisback) [reply helm-chart/install.sh:23]: 手误

File: docker-compose/README.md (hunk 1 above)
Comment (ganisback) [reply docker-compose/README.md:105]: OK

**Review by wayneliu0019 — APPROVED** [review]

**Pull request merged**